This trips a warning which breask the rest of the script,
possibly because the output is a blank line

--- a/setup.py	Sun Apr  2 16:59:20 2023
+++ b/setup.py	Sat Apr  8 16:07:42 2023
@@ -383,9 +383,9 @@
                 if token.startswith("-L"):
                     self.library_dirs.append(token[2:])
 
-            for token in pg_config_helper.query("cppflags").split():
-                if token.startswith("-I"):
-                    self.include_dirs.append(token[2:])
+            #for token in pg_config_helper.query("cppflags").split():
+            #    if token.startswith("-I"):
+            #        self.include_dirs.append(token[2:])
 
             pgversion = pg_config_helper.query("version").split()[1]
 
